test/tests/LayerTest.coffee

```diff
@@ -584,6 +584,27 @@ describe "Layer", ->
 			layer.shadowColor = null
 			layer.style.boxShadow.should.equal "rgba(0, 0, 0, 0) 10px 10px 10px 10px"
 
+		it "should set the shadow via layer.style.boxShadow", ->
+
+			layer = new Layer
+				style: boxShadow: "inset 10px 10px 10px rgba(255, 0, 0, 0.5)"
```

Comment:
1. The declaration of style-properties in the constructor may be against your testing guidelines, but this was done on purpose, as this is what causes the problem. Setting `layer.style.boxShadow` after init works right now, but this changes aren't propagated to Framer's shadow properties, yet. See lines 593-597.

2. I've included the optional `inset`-property as well, as this is the primary reason for why people are using this hack / workaround. Support for the `inset`-property would be desired.

Potential ways to fix these problems:

- Rework [this part of Layer.coffee](https://github.com/koenbok/Framer/blob/master/framer/Layer.coffee#L49-L55). I believe there's a problem with how you guys map CSS properties to `layer.style`: `box-shadow` for instance, gets mapped to `layer.style.boxShadow` and loses the hyphen in the process. Hence why I think the conditional in the code block linked above does not work properly.

- Introduce `layer.shadowInset = true` and include it here: [LayerStyle.coffee](https://github.com/koenbok/Framer/blob/master/framer/LayerStyle.coffee#L160-L169). If you decide to do that, a test for `layer.shadowInset = true` should be added as well.